proto/endpoint_server.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "forch/proto/shared_constants.proto";
+
+/*
+ * Tunnel endpoint params.
+ */
+ message Endpoint {
+    // IP address of tunnel endpoint.
+    string ip = 1;
+  
+    // UDP port for tunnel.
+    int32 port = 2;
+  
+    // Tunnel key index.
+    int32 vni = 3;
+
+    // Port on t1 switch
+    int32 tap_port = 4;
+  }
+
+/*
+ * Device testing session server endpoint.
+ */
+ service EndpointServer {
+     // Configure Vxlan interface
```

Comment:
nit: indent

also, not sure the comment is helpful/accurate -- at this level (where the comment is), the interface may very well not be vxlan.  Personally I'd just remove it -- excess comments often seem like a good idea at the time, but in the long run they end up being outdated.